plugins/api-docs/src/components/ApiExplorerTable/ApiExplorerTable.tsx

```diff
@@ -14,55 +14,90 @@
  * limitations under the License.
  */
 
-import { ApiEntityV1alpha1, Entity } from '@backstage/catalog-model';
+import {
+  ApiEntityV1alpha1,
+  Entity,
+  EntityName,
+  RELATION_OWNED_BY,
+  RELATION_PART_OF,
+} from '@backstage/catalog-model';
 import {
   Table,
   TableColumn,
   TableFilter,
   TableState,
   useQueryParamState,
 } from '@backstage/core';
+import {
+  EntityRefLink,
+  EntityRefLinks,
+  formatEntityRefTitle,
+  getEntityRelations,
+} from '@backstage/plugin-catalog-react';
 import { Chip } from '@material-ui/core';
 import { Alert } from '@material-ui/lab';
 import React from 'react';
 import { ApiTypeTitle } from '../ApiDefinitionCard';
-import { EntityLink } from '../EntityLink';
 
-const columns: TableColumn<Entity>[] = [
+type EntityRow = {
+  entity: ApiEntityV1alpha1;
+  resolved: {
+    partOfSystemRelationTitle?: string;
+    partOfSystemRelations: EntityName[];
+    ownedByRelationsTitle?: string;
+    ownedByRelations: EntityName[];
+  };
+};
+
+const columns: TableColumn<EntityRow>[] = [
   {
     title: 'Name',
-    field: 'metadata.name',
+    field: 'entity.metadata.name',
     highlight: true,
-    render: (entity: any) => (
-      <EntityLink entity={entity}>{entity.metadata.name}</EntityLink>
+    render: ({ entity }) => (
+      <EntityRefLink entityRef={entity}>{entity.metadata.name}</EntityRefLink>
```

Comment:
i just realized - what if it's in a non-default namespace? Maybe it should actually render the text as `my-namespace/the-name` as well? Welp, future extension.

Reply:
Good question, that would be as easy as setting a `defaultKind="API"` and removing the component child.

We should do that

Reply:
I updated it both in the catalog and API explorer, looks like this: 
![image](https://user-images.githubusercontent.com/648527/106487434-ef7f8480-64b2-11eb-97a4-079aed728145.png)